### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by AleksyBalazinski**

### CODE REVIEW

File: abstracts/abstract_en.tex [hunk 1]
```diff
@@ -9,14 +9,17 @@
...
 
-    The game also features a number of classic game mechanics, such as terrain editing, character animations, day and night cycle, and many more.
+    The procedural terrain generation in the game is based on the marching cubes algorithm with Perlin noise used for generating the underlying scalar field.
+
+    The game also includes several classic game features, such as terrain editing, character animations, day/night cycle, and many more.
     As a result, the game offers a unique experience of exploring and interacting with worlds inherently different from the one we live in. \\
 
-    \noindent \textbf{Keywords:} Non-Euclidean Geometry, Hyperbolic Geometry, Spherical Geometry, Procedural Generation, Video Games, OpenGL, C\#
-    \question{Should keywords start with capital letters?}
+    \noindent \textbf{Keywords:} non-Euclidean geometry, hyperbolic geometry, spherical geometry, procedural terrain generation, video games, OpenGL, C\#
```
Comment (KarolDenst) [inline abstracts/abstract_en.tex:23]: If we change to lower case we should also do that in the polish version

File: chapters/functional_specification/functional_specification.tex [hunk 2]
```diff
@@ -1,11 +1,11 @@
 \chapter{Functional Specification}\label{ch:functional_specification}
-The functional requirements are described using use cases.
-The functional requirements are listed in \autoref{tab:functional_requirements}.
+The functional requirements are described in the form of use cases and listed in \autoref{tab:functional_requirements}.
+\todo{Are those use cases or user stories? I never know}
```
Comment (KarolDenst) [inline chapters/functional_specification/functional_specification.tex:3]: user stories. For sure not use cases

File: chapters/implementation/sections/terrain/subsections/scalar_field.tex [hunk 3]
```diff
@@ -1,35 +1,34 @@
...
         \includegraphics[width=0.8\textwidth]{chapters/implementation/sections/terrain/resources/octaves-1.png}
-        \caption{Small octaves (1).}
+        \caption{Small number of octaves (1).}
     \end{subfigure}
     \hfill
     \begin{subfigure}{0.45\textwidth}
         \centering
         \includegraphics[width=0.8\textwidth]{chapters/implementation/sections/terrain/resources/octaves-5.png}
-        \caption{Big octaves (5).}
+        \caption{Big number of octaves (5).}
```
Comment (KarolDenst) [inline chapters/implementation/sections/terrain/subsections/scalar_field.tex:31]: Maybe Large number of ...

File: chapters/implementation/sections/two_dimensional_graphics/subsections/hud.tex [hunk 4]
```diff
@@ -24,7 +24,9 @@ \subsection{Heads Up Display} \label{sec:hud}
 It is drawn at the bottom of the screen.
 It uses both the item's texture containing the HUD items and the texture containing the alphabet.
 The images of the items are drawn first and then the text is drawn on top of them.
-It is done in this order to optimize the performance by minimizing the number of times a texture is set to 2.
+%It is done in this order to optimize the performance by minimizing the number of times a texture is set to 2.
+This way the texture is set only two times, which improves the performance.
+\todo{Can we remove the previous sentence?}
```
Comment (KarolDenst) [inline chapters/implementation/sections/two_dimensional_graphics/subsections/hud.tex:29]: yes

File: chapters/results/sections/non_euclidean/subsections/hyperbolic.tex [hunk 5]
```diff
@@ -28,7 +28,7 @@ \subsection{Hyperbolic space}
 As mentioned in \autoref{subsubsec:practical-considerations-hyperbolic-space} in our approach the camera's position (as passed to the view matrix) is fixed.
 Changing the point where it is located allows us to change the curvature of the terrain.
 \autoref{fig:hyperbolic-space-curvatures} shows the scene in hyperbolic space where the camera is positioned close to the origin (cf. \autoref{fig:hyperbolic-space-small-curvature}) resulting in small curvature as compared to the scene when the camera is moved farther from the origin (cf. \autoref{fig:hyperbolic-space-big-curvature}).
-\begin{figure*}[h]
+\begin{figure*}[!htb]
```
Comment (KarolDenst) [inline chapters/results/sections/non_euclidean/subsections/hyperbolic.tex:31]: why is this figure*?

File: chapters/results/sections/non_euclidean/subsections/hyperbolic.tex [hunk 6]
```diff
@@ -3,7 +3,7 @@ \subsection{Hyperbolic space}
 This may create an illusion that the terrain is wrapped around a giant sphere.
 However, by exploring the hyperbolic space, we can quickly notice that it is infinite, just like the Euclidean space.
 \autoref{fig:hyperbolic-space-games} shows the comparison of the depictions of hyperbolic space between our game and \textit{Hyperbolica}.
-\begin{figure*}[h]
+\begin{figure*}[!htb]
```
Comment (KarolDenst) [inline chapters/results/sections/non_euclidean/subsections/hyperbolic.tex:6]: why is this figure*?

File: chapters/results/sections/non_euclidean/subsections/spherical.tex [hunk 7]
```diff
@@ -2,7 +2,7 @@ \subsection{Spherical space}
 Playing the video game in spherical space gives the impression that the terrain is wallpapered onto the inside of a giant sphere.
 Unlike the hyperbolic space, the spherical space is finite.
 Comparison in \autoref{fig:spherical-space-games} shows that our implementation provides visual effects to a certain degree similar to those in \textit{Hyperbolica}.
-\begin{figure*}[h]
+\begin{figure*}[!htb]
```
Comment (KarolDenst) [inline chapters/results/sections/non_euclidean/subsections/spherical.tex:5]: why is this figure*?

File: chapters/results/sections/non_euclidean/subsections/spherical.tex [hunk 8]
```diff
@@ -28,7 +28,7 @@ \subsection{Spherical space}
 Furthermore, unlike in Euclidean geometry, objects further from the player do not always appear smaller, sometimes they can even appear larger.
 This effect of "reversed perspective" can be seen in both games as shown in \autoref{fig:spherical-space-reversed-perspective}.
 In \autoref{fig:spherical-space-reversed-perspective-hyper}, the car and the bot to the right of the car's hood are further from the camera than other objects in the scene.
-\begin{figure*}[h]
+\begin{figure*}[!htb]
```
Comment (KarolDenst) [inline chapters/results/sections/non_euclidean/subsections/spherical.tex:31]: why is this figure*?

Comment (KarolDenst) [reply chapters/results/sections/non_euclidean/subsections/spherical.tex:31]: There are to may to comment so I'll just ask here. Why are some figure* and some figure?

File: abstracts/abstract_en.tex [hunk 9]
```diff
@@ -9,14 +9,17 @@
...
     This work describes the implementation of a video game that incorporates both of the aforementioned concepts.
 
-    In the game a technique of transforming Euclidean space into a hyperbolic or spherical space was used, which allows for the use of established algorithms.
-    This approach simplifies the implementation of the game logic in different geometries as it allows sharing large parts of the implementation across the three geometries.
-    It also allows for changing the degree to which the scene is visually curved in hyperbolic geometry at runtime, which is the best way to showcase the differences between the geometries.
+    In the game, we use and expand on the technique of transforming Euclidean space into hyperbolic or spherical space introduced by László Szirmay-Kalos and Milán Magdics in \cite{Szirmay-Kalos2022}.
+    The approach allows for the use of established algorithms, developed with Euclidean geometry in mind, in other geometries.
+    %This approach also simplifies the implementation of the game logic in different geometries as large parts of the implementation can be shared across the three geometries.
```
Comment (KarolDenst) [inline abstracts/abstract_en.tex:14]: I think this can be removed

File: abstracts/abstract_pl.tex [hunk 10]
```diff
@@ -4,7 +4,7 @@
     \begin{center}
         \tytul
     \end{center}
-
+    \todo{Przetłumaczyć}
```
Comment (KarolDenst) [inline abstracts/abstract_pl.tex:7]: I think the english version looks good

File: chapters/introduction/introduction.tex [hunk 11]
```diff
@@ -11,27 +11,28 @@ \chapter{Introduction}\label{ch:introduction}
 It allows for creating virtually infinite worlds that are unique in every playthrough.
```
Comment (KarolDenst) [inline chapters/introduction/introduction.tex:11]: Think we can remove lines 6 from this file

File: abstracts/abstract_en.tex (hunk 9 above)
Comment (AleksyBalazinski) [reply abstracts/abstract_en.tex:14]: Do you mean the commented-out part only?

File: abstracts/abstract_pl.tex (hunk 10 above)
Comment (AleksyBalazinski) [reply abstracts/abstract_pl.tex:7]: translated

File: chapters/results/sections/non_euclidean/subsections/spherical.tex (hunk 8 above)
Comment (AleksyBalazinski) [reply chapters/results/sections/non_euclidean/subsections/spherical.tex:31]: Honestly idk I copy-pasted this from latex stack exchange

Comment (AleksyBalazinski) [reply chapters/results/sections/non_euclidean/subsections/spherical.tex:31]: image

Comment (AleksyBalazinski) [reply chapters/results/sections/non_euclidean/subsections/spherical.tex:31]: but when i change it it seems like it doesnt matter, works either way

Comment (AleksyBalazinski) [reply chapters/results/sections/non_euclidean/subsections/spherical.tex:31]: should I change this to `figure`?

File: abstracts/abstract_en.tex (hunk 9 above)
Comment (KarolDenst) [reply abstracts/abstract_en.tex]: yes

File: abstracts/abstract_pl.tex [hunk 12]
```diff
@@ -4,19 +4,21 @@
...
+    Zastosowana metoda pozwala również na zmianę stopnia zakrzywienia sceny w czasie rzeczywistym w przestrzeni hiperbolicznej.
+
+    Proceduralne generowanie terenu w grze opiera się na algorytmie maszerujących sześcianów (ang. \textit{marching cubes algorithm}) z polem skalarnym określonym za pomocą szumu Perlina.
 
-    Gra zawiera również wiele klasycznych mechanik gier, takich jak edycja terenu, animacje postaci, cykl dnia i nocy oraz wiele innych.
+    Gra zawiera również wiele klasycznych funkcjonalności, takich jak edycja terenu, animacje postaci, cykl dnia i nocy oraz wiele innych.
     W rezultacie gra oferuje unikalne doświadczenie eksploracji i interakcji z światami fundamentalnie różnymi od tego, w którym żyjemy. \\
 
-    \noindent \textbf{Słowa kluczowe:} Geometria Nieeuklidesowa, Geometria Hiperboliczna, Geometria Sferyczna, Generacja Proceduralna, Gry Wideo, OpenGL, C\#
+    \noindent \textbf{Słowa kluczowe:} geometria nieeuklidesowa, geometria hiperboliczna, geometria sferyczna, proceduralne generowanie terenu, gry wideo, OpenGL, C\#
```
Comment (KarolDenst) [inline abstracts/abstract_pl.tex:22]: gry wideo -> gry komputerowe

**Review by KarolDenst — APPROVED** [review]

**Pull request merged**